devel/libtatsu/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        libimobiledevice libtatsu 1.0.4
+revision            0
+github.tarball_from archive
+
+categories          devel
+
+license             LGPL-2.1
+maintainers         {gmail.com:turbocool3r @turbocool3r} openmaintainer
+
+description         Library handling the communication with Apple's Tatsu Signing Server (TSS).
+
+long_description    This library is part of the libimobiledevice project. The main purpose of this \
+                    library is to allow creating TSS request payloads, sending them to Apple's TSS \
+                    server, and retrieving and processing the response. \
+                    \
+                    The code was originally part of idevicerestore and has been split out to its own \
+                    library, while obviously idevicerestore will remain the main consumer.
+
+homepage            https://www.libimobiledevice.org/
+
+checksums           rmd160  4f650c82b522d5af8c3a65fee89ad0c5a25252fc \
+                    sha256  a6169aa678a7404757ac2bc73fe049df73c891a699541280b4336d9bd7794506 \
+                    size    28393
+
+depends_build-append \
+                    port:autoconf \
+                    port:automake \
+                    port:libtool \
+                    port:pkgconfig
+
+depends_lib         port:libplist
+
+pre-configure {
+    system -W ${worksrcpath} "echo ${version} > .tarball-version"
+}
+configure.cmd       ./autogen.sh
+# this overrides pkg-config's libcurl choice like README requests
+configure.env-append \
+                    libcurl_CFLAGS=${configure.sdkroot}/usr/include \
+                    libcurl_LIBS=-lcurl
+
+subport libtatsu-devel {
+    github.setup    libimobiledevice libtatsu 7e1647b9883ff1daa6363de20af2c4129ed45dcd
+    version         20241203
+    revision        0
+
+    checksums       rmd160  4f650c82b522d5af8c3a65fee89ad0c5a25252fc \
+                    sha256  a6169aa678a7404757ac2bc73fe049df73c891a699541280b4336d9bd7794506 \
+                    size    28393
+
+    conflicts       libtatsu
+
+    depends_lib-replace \
+                    port:libplist \
+                    port:libplist-devel
+
+    livecheck.url   ${github.homepage}/commits/${github.livecheck.branch}.atom
+}
+
+if {${subport} eq ${name}} {
+    conflicts       libirecovery-devel
```

Comment:
shouldn't this be "libtatsu-devel"? Also, I don't think it doesn't need to be in this block, it can just go up close to the "name" field - right?

Reply:
thanks, missed that one. do you mean I should move it up in the file? that part wasn't changed a lot from the libirecovery portfile, libplist portfile https://github.com/macports/macports-ports/blob/master/textproc/libplist/Portfile seems to do this in a similar way. I'm open to changes though.

Reply:
Pushed the name fix for now.

Reply:
yes, move it up to under the "name" field. Double check that it works correctly, but it should

Reply:
moved it up, but now for some reason my local port lint is unhappy about libtatsu-devel port not being known. CI seems to like everything.